File: ejercicios/src/main/java/co/edu/sena/ejercicios/ejercicio03/Consola.java
package co.edu.sena.ejercicios.ejercicio03;

public class Consola {
    private int controles;
    private String conectorSonido;
    private String conectorVideo;

    public Consola(int controles, String conectorSonido, String conectorVideo) {
        this.controles = controles;
        this.conectorSonido = conectorSonido;
        this.conectorVideo = conectorVideo;
    }

    public int getControles() {
        return controles;
    }

    public void setControles(int controles) {
        this.controles = controles;
    }

    public String getConectorSonido() {
        return conectorSonido;
    }

    public void setConectorSonido(String conectorSonido) {
        this.conectorSonido = conectorSonido;
    }

    public String getConectorVideo() {
        return conectorVideo;
    }

    public void setConectorVideo(String conectorVideo) {
        this.conectorVideo = conectorVideo;
    }
}

File: typescript/src/ejercicio03/consola.ts
export abstract class Consola {
    private _controles: number;
    private _conectorSonido: string;
    private _conectorVideo: string;

    protected constructor(controles: number, conectorSonido: string, conectorVideo: string) {
        this._controles = controles;
        this._conectorSonido = conectorSonido;
        this._conectorVideo = conectorVideo;
    }

    get controles(): number {
        return this._controles;
    }

    set controles(value: number) {
        this._controles = value;
    }

    get conectorSonido(): string {
        return this._conectorSonido;
    }

    set conectorSonido(value: string) {
        this._conectorSonido = value;
    }

    get conectorVideo(): string {
        return this._conectorVideo;
    }

    set conectorVideo(value: string) {
        this._conectorVideo = value;
    }
}